client/src/components/InputField/InputField.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
```

Comment:
`classnames` library has already been committed in the previous commits.

Reply:
will take from the master while merging